package.json

```diff
@@ -121,8 +121,8 @@
     "typescript": "3.3.x",
     "uglify-es": "^3.3.9",
     "webpack": "^4.25.1",
-    "webpack-cli": "^3.1.2",
-    "webpack-dev-server": "^3.1.14",
+    "webpack-cli": "^3.3.11",
+    "webpack-dev-server": "^3.11.0",
```

Comment:
These seem like big upgrades which may be outside the scope of the story - do they require regression on the app? are there any release notes between those versions?

Reply:
I didn't intend for those updates to be made. They are removed now.